cfg/eclipse-keys.lua

```diff
@@ -0,0 +1,100 @@
+-- In your user / system configuration file simply type: include "eclipse-keys.lua"
```

Comment:
Seems to work, based on [this](https://studio.zerobrane.com/doc-configuration#configuration-file-commands) document